stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -133,10 +134,12 @@ public static InspectorModulesProvider defaultInspectorModulesProvider(final Con
       public Iterable<ChromeDevtoolsDomain> get() {
         ArrayList<ChromeDevtoolsDomain> modules = new ArrayList<ChromeDevtoolsDomain>();
         modules.add(new Console());
-        modules.add(new CSS());
         modules.add(new Debugger());
         if (Build.VERSION.SDK_INT >= AndroidDOMConstants.MIN_API_LEVEL) {
-          modules.add(new DOM(new AndroidDOMProviderFactory((Application)context.getApplicationContext())));
+          Document doc = new Document(new AndroidDOMProviderFactory((Application)context.getApplicationContext()));
+
+          modules.add(new DOM(doc));
+          modules.add(new CSS(doc));
```

Comment:
This makes it so that the CSS module also requires the same minimum API level as DOM. @jasta should comment on this too.